py3dbp/main.py

```diff
@@ -1,5 +1,12 @@
 from .constants import RotationType, Axis
 from .auxiliary_methods import intersect, set_to_decimal
+from copy import deepcopy
+
+
+# required to plot a representation of Bin and contained items
+import matplotlib.pyplot as plt
```

Comment:
Remove plot function and dependencies from this main package. Create this functions outside, keep packer classes clean.

Reply:
Moved to auxiliary_methods.py